cobra/test/test_flux_analysis/conftest.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+
+"""Define module level fixtures."""
+
+
+from __future__ import absolute_import
+
+from pandas import Series
+
+import cobra.util.solver as sutil
+import pytest
+from cobra.core import Metabolite, Model, Reaction, Solution
+
+# The scipy interface is currently unstable and may yield errors or infeasible
+# solutions.
+
+
+@pytest.fixture(scope="session",
+                params=[s for s in ["glpk", "cplex", "gurobi"]
+                        if s in sutil.solvers])
+def all_solvers(request):
+    """Return the avaialble solvers."""
+    return request.param
+
+
+@pytest.fixture(scope="session",
+                params=[s for s in ["cplex", "gurobi"]
+                        if s in sutil.solvers])
+def qp_solvers(request):
+    """Return the available QP solvers."""
+    return request.param
+
+
+@pytest.fixture(scope="session")
```

Comment:
Please move this fixture into the `test_room` module and change the scope to `"module"`.

Reply:
Sure!

Reply:
I totally forgot to mention that `room_model` and `room_solution` fixtures are needed in `test_deletion.py` too. Will it be okay to keep these fixtures as they are?

Reply:
Okay, then keep them in the `conftest.py` but still change the scope to `"module"`, please.

Reply:
Done!